doc/man7/provider-mac.pod

```diff
@@ -211,6 +214,16 @@ With some MAC algorithms, it can also be used to set the size that the
 resulting MAC should have.
 Allowable sizes are decided within each implementation.
 
+=item "c-rounds" (B<OSSL_MAC_PARAM_C_ROUNDS>) <unsigned integer>
+
+Specifies the number of rounds per message block for SipHash.  By default this
+is I<2>.
+
+=item "d-rounds" (B<OSSL_MAC_PARAM_D_ROUNDS>) <unsigned integer>
+
+Specifies the number of finalisation rounds for SipHash.  By default this is
+I<4>.
+
```

Comment:
Why are you adding these here, especially since you already added them in EVP_MAC-SipHash.pod???

Reply:
Should we not document this?

Reply:
Yes.  Which you have done in EVP_MAC-SipHash.pod.  This is a misplaced duplicate

Reply:
There are plenty more cases of similar.  I'm happy to remove this but it goes against the grain.

Reply:
We document _key_, _IV_, _custom_, _salt_, _block XOF_, _digest no-init_, _digest-one-shot_ in this man page.  Why not _crounds_ and _drounds_?

Please elucidate as to a consequential difference between the former and the latter?

Reply:
I actually think those are generally misplaced in this particular file, *unless* they are parameters that are expected to be handled by *all* MAC implementations without exception.

The idea with `provider-mac.pod` is to be a manual for *provider authors* (there's even a note about that at the beginning of DESCRIPTION), and are therefore meant to be generic rather than a description of *our* providers, even to the point of not describing individual implementations.

The idea is that *our* providers and *our* implementations are described in other manuals dedicated to them.  So the description of *our* default provider is in `OSSL_PROVIDER-default.pod`, and the description of *our* SipHash implementation is in `EVP_MAC-SipHash.pod`.

In essence, it's about very different audiences:
-   `provider-{name}.pod` are for provider *authors*
-   `{METHODNAME}-[AlgorithmName}.pod` are for provider *users*

[I have said this a number of times]

Reply:
I've removed these two from the file.